sdk/core/core-http/lib/policies/proxyPolicy.ts

```diff
@@ -13,22 +13,93 @@ import { WebResource } from "../webResource";
 import { Constants } from "../util/constants";
 import { URLBuilder } from "../url";
 
+function environmentProxyValue(name: string): string | undefined {
```

Comment:
Seems like this is generic enough to be something like getEnvironmentValue and used in other places.

Reply:
Yes, I used it in all places which need to get proxy environment variable.